Makefile

```diff
@@ -26,6 +26,9 @@ $(DEPS_DIR)/rabbit:
 	git clone https://github.com/rabbitmq/rabbitmq-server.git $@
 	$(MAKE) -C $@ fetch-deps DEPS_DIR="$(abspath $(DEPS_DIR))"
 
+$(DEPS_DIR)/rabbitmq_ct_helpers:
+	git clone https://github.com/rabbitmq/rabbitmq-ct-helpers.git $(DEPS_DIR)/rabbitmq_ct_helpers
+
```

Comment:
You can use the `$@` Make variable to reduce repetition:
```
git clone https://github.com/rabbitmq/rabbitmq-ct-helpers.git "$@"
```

(plus the quotes to protect against spaces)